pkg/apis/servicecatalog/v1alpha1/types.go

```diff
@@ -189,12 +211,15 @@ type Instance struct {
 	Status InstanceStatus `json:"status"`
 }
 
-// InstanceSpec represents a description of an Instance.
+// InstanceSpec represents the desired state of an Instance.
 type InstanceSpec struct {
-	// ServiceClassName is the reference to the ServiceClass this is an
-	// instance of.  Immutable.
+	// ServiceClassName is the reference to the ServiceClass this Instance
+	// should be provisioned from.
+	// Immutable.
```

Comment:
I think this is going to smash up #854 pretty good.

If you both feel `Immutable` needs to be on it's own line, add an empty comment line. I've seen plenty of formatters smash that back up as if it needs to be tacked on to the end of the previous paragraph.